lizzy/job/deployer.py

```diff
@@ -113,7 +113,11 @@ def deployed(self):
             except ExecutionError:
                 self.logger.exception("Failed to switch app traffic.", extra=self.log_info)
 
-        all_versions = sorted(self.lizzy_stacks[self.stack.stack_name].keys())
+        all_versions = sorted(
+            self.lizzy_stacks[self.stack.stack_name].values(),
+            key=lambda s: s.creation_time)
+        all_versions = [stack.stack_name for stack in all_versions]
```

Comment:
Sure, I thought about that but since it haven't changed the final content I kept as it was before.